src/blas/backends/rocblas/rocblas_task.hpp

```diff
@@ -29,9 +29,7 @@
 #include <CL/sycl/detail/pi.hpp>
 #else
 #include "rocblas_scope_handle_hipsycl.hpp"
-namespace cl::sycl {
-using interop_handler = cl::sycl::interop_handle;
```

Comment:
Could you please explain the reason for this change?

Reply:
I believe this is a reminiscence of some intermittent development stage in the rocBLAS backend. This change is not necessary, however, it only complicates the code unnecessarily. The original purpose was probably to have a common interface for both dpc++ and hipSYCL, however we use ifdefs now. Therfore it is not necessary to have both `interop_handle` and `interop_handler` available when compiling with hipSYCL.

Reply:
Would you prefer not to remove these few lines, since they are not strictly related to this PR?

Reply:
yes please